src/Form/Type/ChoiceFieldMaskType.php

```diff
@@ -62,7 +63,7 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getParent()
     {
-        return 'choice';
+        return ChoiceType::class;
```

Comment:
afaik this does not work with symfony 2.8, or does it? if it does not, lets raise the minimal symfony version to 3.4 to keep things manageable.

Reply:
are you sure that this works with symfony 2?